### DISCUSSION TITLE
Pregunta sobre fecha de entrega TPI: File System

### DISCUSSION THREAD

**Discussion by akalinin11** [dds-jv/foro#369] [Consultas generales] [answered]
Hola buenos días! Estaba revisando la fecha de entrega de este trabajo y dice 29 de mayo pero la misma figura para la clase de 09/06.
Desde ya muchas gracias, Saludos!

**Comment by gastonprieto** [comment, marked as answer]
buenas @akalinin11 

Si, estaba medio raro, la fecha original era para el 29/5, pero dado la confusión, agarra y la cambie para el 04/06, asi si quieren mandar actualizaciones pueden hacerlo

**Reply by AlejoGol** [reply]
Buenas, entonces a pesar de que según el calendario la fecha en la que el tp se daría de tarea es el 09/06 la fecha de entrega es el 04/06?

**Reply by gastonprieto** [reply]
antes que nada, que curso sos @AlejoGol ? viernes o jueves?

**Reply by AlejoGol** [reply]
jueves

**Reply by gastonprieto** [reply]
si, es el fin de semana del 4/6, la fecha es la q decia entre parentesis, originalmente era el 29/5 asi que por la confusion la pasamos una semana mas
